CHANGELOG.md

```diff
@@ -311,6 +311,7 @@
 * [BUGFIX] Ingester: increment series per metric when recovering from WAL or transfer. #2674
 * [BUGFIX] Fixed `wrong number of arguments for 'mget' command` Redis error when a query has no chunks to lookup from storage. #2700 #2796
 * [BUGFIX] Ingester: Automatically remove old tmp checkpoints, fixing a potential disk space leak after an ingester crashes. #2726
+* [BUGFIX] Improve the error message in readiness check when unhealthy ingesters are in the ring. #3158
```

Comment:
This should go to “master” version, not 1.2.

(Also not sure if this needs to have a changelog entry)

Reply:
what changelog entry 😉